processor/configuration/src/main/java/com/redhat/service/smartevents/processor/actions/slack/SlackConnectorAction.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.service.smartevents.processor.actions.slack;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.redhat.service.smartevents.processor.actions.generic.GenericConnectorAction;
+
+@ApplicationScoped
+public class SlackConnectorAction extends GenericConnectorAction {
```

Comment:
Brainstorming: we could remove all the connector specific classes with the following workflow: 
1) We get a connector action
2) if the `type` is for example `slack_sink_0.1`, we look for the `slack_sink_0.1.json` file
3) we use `slack_sink_0.1.json` for the validation

In this way we can
1) add a new connector by just adding the json in the `resources` folder
2) no need to write a single line of code for any connector

On the other side, if we keep connector-specific classes we might add additional checks on the payload. For example if a parameter is an URL, the json schema validation would just check that it is a string, but we could also check that the value is a valid URL.